[DBM][Postgres] Removing low traffic instructions (#31343)
DataDog/documentation: content/en/database_monitoring/setup_postgres/selfhosted.md

## Install the Agent

Installing the Datadog Agent also installs the Postgres check, which is required for Database Monitoring on Postgres. 

> If you haven’t installed the Agent, see the [Agent installation instructions][8]. Then, return here to continue with the instructions for your installation method.

Reviewer: ```suggestion
If you haven't installed the Agent, see the [Agent installation instructions][8]. Then, return here to continue with the instructions for your installation method.
```

> If you haven't installed the Agent yet, see the [Agent installation instructions][8], then return here and follow the instructions below based on how you installed the Agent.

Reviewer: ```suggestion
If you haven’t installed the Agent, see the [Agent installation instructions][8]. Then, return here to continue with the instructions for your installation method.
```

> Edit the Agent's `conf.d/postgres.d/conf.yaml` file to point the Postgres instance you want to monitor. For a complete list of configuration options, see the [sample postgres.d/conf.yaml][1].

Reviewer: ```suggestion
Edit the Agent's `conf.d/postgres.d/conf.yaml` file to point to the Postgres instance you want to monitor. For a complete list of configuration options, see the [sample postgres.d/conf.yaml][1].
```
